src/ontology/mondo-edit.obo

```diff
@@ -440512,6 +440512,18 @@ is_a: MONDO:0700064 {source="https://orcid.org/0000-0002-4142-7153"} ! aneuploid
 relationship: has_modifier SO:0000055 {source="https://orcid.org/0000-0002-4142-7153"} ! hyperploid
 created_by: http://orcid.org/0000-0002-4142-7153
 
+[Term]
+id: MONDO:0800001
+name: delayed sleep phase syndrome, susceptibility to
+def: "An inherited susceptibility or predisposition to developing delayed sleep phase syndrome." [MONDO:patterns/inherited_susceptibility]
+xref: OMIM:614163 {source="MONDO:equivalentTo"}
+is_a: MONDO:0020573 {source="OMIM:614163"} ! inherited disease susceptibility
```

Comment:
@rays22 this looks great, nice work!

I forgot to mention that we add source annotations to all of the subclassOf assertions in Mondo. I'll add this to our documentation. I went ahead and added this. 

Thanks for your contribution!